assets/js/field-slug.js

```diff
@@ -140,7 +140,9 @@ class Slugger {
      */
     listenTarget() {
         for (const target of this.targets) {
-            target.addEventListener('keyup', () => {
+            const eventType = target.tagName === 'SELECT' ? 'change' : 'keyup';
+
+            target.addEventListener(eventType, () => {
```

Comment:
I think the `change` event can just be replaced by an `input` event:
```suggestion
            target.addEventListener('input', () => {
```
See: https://developer.mozilla.org/docs/Web/API/Element/input_event

Reply:
Oh yeah, you're right! Thanks 💯